posthog/api/integration.py

```diff
@@ -27,7 +29,17 @@ def create(self, validated_data: Any) -> Any:
         request = self.context["request"]
         team_id = self.context["team_id"]
 
-        if validated_data["kind"] in OauthIntegration.supported_kinds:
+        if validated_data["kind"] in GoogleCloudIntegration.supported_kinds:
```

Comment:
This is very picky but I don't get why we do file attachements here versus just having the data on the payload. The file is just a json config right?

Reply:
The file itself is supposed to be a "secret json key" of sorts. I'm not sure where I'd put it in the payload. It should feed the "integration" system, and be kept alive through a token refresh job. Adding this into the Hog function itself would require us to implement some form of scheduling within Hog. I _could_'ve made a popup that asks for you to copy past the key file's contents when clicking "new gcloud integration", but that wouldn't have made anything better.